pom.xml

```diff
@@ -43,29 +51,17 @@
       <dependency>
         <groupId>io.micrometer</groupId>
         <artifactId>micrometer-bom</artifactId>
-        <version>1.10.3</version>  <!-- https://github.com/vert-x3/vertx-micrometer-metrics/blob/master/pom.xml#L41 -->
+        <version>1.11.4</version>  <!-- https://github.com/vert-x3/vertx-micrometer-metrics/blob/master/pom.xml#L41 -->
         <type>pom</type>
         <scope>import</scope>
       </dependency>
       <dependency>
         <groupId>com.hazelcast</groupId>
-        <artifactId>hazelcast-aws</artifactId>
-        <version>3.4</version>  <!-- https://github.com/hazelcast/hazelcast-aws#requirements -->
-      </dependency>
-      <dependency>
-        <groupId>com.hazelcast</groupId>
-        <artifactId>hazelcast-kubernetes</artifactId>
-        <version>2.2.3</version>  <!-- https://github.com/hazelcast/hazelcast-kubernetes#requirements-and-recommendations -->
+        <artifactId>hazelcast</artifactId>
+        <version>4.2.8</version>  <!-- https://github.com/vert-x3/vertx-hazelcast/blob/4.4/pom.xml#L19 -->
```

Comment:
Can we go to hazlecast 5 @julianladisch? I'm looking at @hjiebsco's PR here: https://github.com/folio-org/okapi/pull/1323. From the looks of this it vertx-hazelcast has been upgraded to handle hazelcast 5 here: https://github.com/vert-x3/vertx-hazelcast/pull/184.

Reply:
Looks like you were making the change just as I was typing the comment.